data/generate_ember_co2.js

```diff
@@ -0,0 +1,55 @@
+const fs = require("fs");
+let csv = fs.readFileSync("data/co2-intensities-ember-2021.csv");
+
+const array = csv.toString().split("\n");
+
+/* Store the converted result into an array */
+const csvToJsonResult = {};
+const gridIntensityResults = {}
+
+/* Store the CSV column headers into seprate variable */
+const headers = array[0].split(",");
+
+/* Iterate over the remaining data rows */
+for (let i = 1; i < array.length - 1; i++) {
```

Comment:
You have used this form of looping elsewhere in the file:

```js
for (let item of currentArray) { 
  doStuff(item) 
}
```

But not here. Is there an advantage to using older C-style loop here?